doc/release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,80 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+These release notes support the [Eclipse OpenJ9 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.8/plan).
+
+
+## Supported platforms
+
+OpenJDK Version 8 binaries containing the Eclipse OpenJ9 virtual machine are supported with this release.
+
+These binaries are available from the [AdoptOpenJDK community](https://adoptopenjdk.net/?variant=openjdk8-openj9).
+Builds are tested against the OpenJDK test suite.
```

Comment:
Can we change this sentence to something like: `Builds are tested against the OpenJ9 FV test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.`

I would rather put the emphasis on OpenJ9 and Adopt.